src/App.tsx

```diff
@@ -1,26 +1,153 @@
 /* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import React, { useEffect, useRef, useState } from 'react';
+import { USER_ID, deleteTodo, getTodos, postTodo } from './api/todos';
 
-const USER_ID = 0;
+import { Status } from './types/Status';
+import { TodoList } from './components/TodoList/TodoList';
+import { Footer } from './components/Footer/Footer';
+import { Errors } from './components/Errors/Errors';
+import { Header } from './components/Header/Header';
+import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [error, setError] = useState('');
+  const [status, setStatus] = useState(Status.All);
+  const [query, setQuery] = useState('');
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [loadingTodos, setLoadingTodos] = useState<number[]>([]);
```

Comment:
```suggestion
  const [loadingTodosCount, setLoadingTodos] = useState<number[]>([]);
```
if its number it should have count